Client/Frontend/Browser/Playlist/Controllers/PlaylistFolderController.swift

```diff
@@ -162,17 +225,22 @@ extension PlaylistFolderController: UITableViewDelegate {
   private func deleteFolder(folder: PlaylistFolder) {
     PlaylistManager.shared.delete(folder: folder)
 
-    do {
-      try self.othersFRC.performFetch()
-    } catch {
-      log.error("Error: \(error)")
-    }
+    Section.allCases.forEach({
+      do {
+        try frc(for: $0)?.performFetch()
+      } catch {
+        log.error("Error: \(error)")
+      }
+    })
 
     tableView.reloadData()
   }
 
   private func onEditFolder(folderUUID: String) {
-    guard let folder = self.othersFRC.fetchedObjects?.first(where: { $0.uuid == folderUUID }) else {
+    guard let folder = Section.allCases
```

Comment:
This is quite complicated to understand actually with 2 different sections with shared and not shared also a default folder with saved. I really think this part should be the responsibility of `PlaylistFolder`.

Reply:
I need it here because the controller needs to decide whether or not a folder can even show. Plus this folder will also have different menus potentially, etc.

This folder can also behave different from other folders since it is syncable and others aren't. Saved also can't be deleted.

So the three different kinds of folders need to handle here, each with their own FRC.